docs/api.md

```diff
@@ -237,6 +237,10 @@ Following is the supported API format for specifying connection tracking:
                  endConnectionTimeout: duration of time to wait from the last flow log to end a connection
                  heartbeatInterval: duration of time to wait between heartbeat reports of a connection
          maxConnectionsTracked: maximum number of connections we keep in our cache (0 means no limit)
+         tcpFlags: settings for handling TCP flags
+             fieldName: name of the field containing TCP flags
+             detectEndConnection: detect end connections by FIN_ACK flag
+             swapAB: swap source and destination when the first flowlog contains the SYN_ACK flag
```

Comment:
I wonder if these two should be the default behavior as it's more reliable & convenient than the timeouts

Reply:
The `swapAB` feature isn't related to the timeouts.
The `detectEndConnection` feature is in addition to the timeouts. It can't replace the timeouts because it's not guaranteed that will receive a flowlog with `FIN_ACK` flag for every TCP connection (either because of sampling or because of SYN attack). But, it may allow us to increase the `endConnectionTimeout` for TCP.